iceoryx_posh/include/iceoryx_posh/internal/runtime/ipc_interface_base.hpp

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) 2019, 2021 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
```

Comment:
i see still some "message queues" in here (e.g. `cleanupOutdatedMessageQueue`). Do you want to refactor that in this PR too or is it better to split it up for smaller PRs?

Reply:
Good catch, thanks. Fixed.